backoff.go

```diff
@@ -0,0 +1,27 @@
+package dsync
+
+import (
+	"math"
+	"time"
+)
+
+// A BackoffFunc determines the backoff strategy for the associated remote call,
```

Comment:
Run this code through goling. Start the comment with the name of the identifier.